tokio-macros/src/entry.rs

```diff
@@ -206,7 +206,12 @@ fn parse_knobs(
                     let msg = "Must have specified ident";
                     return Err(syn::Error::new_spanned(namevalue, msg));
                 }
-                match ident.unwrap().to_string().to_lowercase().as_str() {
+                match ident
+                    .expect("Must have specified ident")
+                    .to_string()
+                    .to_lowercase()
+                    .as_str()
+                {
```

Comment:
Here it would be better to eliminate the unwrap/expect entirely by properly talking to the if right above it.